contracts/modules/cover/Quotation.sol

```diff
@@ -207,51 +185,43 @@ contract Quotation is Iupgradable {
    * @dev Verifies signature.
    * @param coverDetails details related to cover.
    * @param coverPeriod validity of cover.
-   * @param smaratCA smarat contract address.
+   * @param contractAddress smart contract address.
    * @param _v argument from vrs hash.
    * @param _r argument from vrs hash.
    * @param _s argument from vrs hash.
    */
-  function verifySign(
+  function verifySignature(
     uint[] memory coverDetails,
     uint16 coverPeriod,
-    bytes4 curr,
-    address smaratCA,
+    bytes4 currency,
+    address contractAddress,
     uint8 _v,
     bytes32 _r,
     bytes32 _s
-  )
-  public
-  view
-  returns (bool)
-  {
-    require(smaratCA != address(0));
-    require(pd.capReached() == 1, "Can not buy cover until cap reached for 1st time");
-    bytes32 hash = getOrderHash(coverDetails, coverPeriod, curr, smaratCA);
+  ) public view returns (bool) {
+    require(contractAddress != address(0));
+    bytes32 hash = getOrderHash(coverDetails, coverPeriod, currency, contractAddress);
     return isValidSignature(hash, _v, _r, _s);
   }
 
   /**
    * @dev Gets order hash for given cover details.
    * @param coverDetails details realted to cover.
    * @param coverPeriod validity of cover.
-   * @param smaratCA smarat contract address.
+   * @param contractAddress smart contract address.
```

Comment:
????  what's wrong with Smarat? did you leave this review?

https://www.tripadvisor.com/ShowUserReviews-g304551-d456058-r6527063-Hotel_Samrat-New_Delhi_National_Capital_Territory_of_Delhi.html